docs/styleguide.md

```diff
@@ -2,9 +2,9 @@
 
 From the GitHub Style Guide:
 
-> Words are an important part of how software works. Just as we have a style guide for our code, we have a style guide for our tone and our voice. Even though there may be dozens of people creating a product, it should still sound like we speak in one consistent voice.
+> Words are an important part of how the software works. Just as we have a style guide for our code, we have a style guide for our tone and our voice. Even though there may be dozens of people creating a product, it should still sound like we speak in one consistent voice.
```

Comment:
Thanks for your contribution! In this scenario, the word "software" implies not just the actual code comprising of software, but also the development, usage, maintenance etc. Also, we are not talking of a single piece of software to necessitate using "the". The existing form of "how software works" is correct :)